raw-compare.js

```diff
@@ -23,7 +23,7 @@ const observationFieldsToIgnore = [];
 const practitionerFieldsToIgnore = ["resource.meta.versionId", "resource.gender"];
 const procedureRequestFieldsToIgnore = [];
 const diagnosticReportFieldsToIgnore = [];
-const specimenFieldsToIgnore = [];
+const specimenFieldsToIgnore = ["resource.status", "resource.receivedTime", "resource.identifier"];
```

Comment:
These fields will be ignored in comparison:
- status field is not copied
- received time is difficult to include into comparison as it is prone to errors due to timestamp. collectedDateTime field is included istead.
- identifier is also not included following other places